controllers/platformnetwork_controller.go

```diff
@@ -1325,3 +1333,59 @@ func (r *PlatformNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&starlingxv1.PlatformNetwork{}).
 		Complete(r)
 }
+
+// Verify whether we have annotation restore-in-progress
+func (r *PlatformNetworkReconciler) checkRestoreInProgress(instance *starlingxv1.PlatformNetwork) bool {
+	restoreInProgress, ok := instance.Annotations[cloudManager.RestoreInProgress]
+	if ok && restoreInProgress != "" {
+		return true
+	}
+	return false
+}
+
+// Update status and clear annoatation
+func (r *PlatformNetworkReconciler) RestorePlatformNetworkStatus(instance *starlingxv1.PlatformNetwork) error {
+	annotation := instance.GetObjectMeta().GetAnnotations()
+	if annotation != nil {
```

Comment:
Already have it checked in line1252, no need to check again. Only need to restore the status in this function

Reply:
Removed the check